cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpResourceConnector.java

```diff
@@ -73,36 +68,8 @@ public String getDBStackTemplate() throws TemplatingNotSupportedException {
     }
 
     @Override
-    public List<CloudResourceStatus> upscale(AuthenticatedContext auth, CloudStack stack, List<CloudResource> resources) {
```

Comment:
why we removed this?

Reply:
I moved this to abstract

Reply:
I have reused the abstract method. The upscale is almost the same, the only difference is the disk reattachment. I refactored the disk reattachment into a protected method and this method can easily turn off with an override. 

However I have checked, the Azure use a different implementation, the AWS and GCP use the same implementation included the disk reattachment